pkg/awsvpc/parser.go

```diff
@@ -291,6 +300,49 @@ func (rc *AWSresourcesContainer) getSGconfig(
 	return nil
 }
 
+func (rc *AWSresourcesContainer) getIgwConfig(
+	res *vpcmodel.MultipleVPCConfigs,
+	skipByVPC map[string]bool) {
+	for _, igw := range rc.InternetGWList {
+		igwID := *igw.InternetGatewayId
+		igwName := igwID
+		if len(igw.Attachments) != 1 {
+			logging.Warnf("skipping internet gateway %s - it has %d vpcs attached\n", igwName, len(igw.Attachments))
+			continue
+		}
+		vpcUID := *igw.Attachments[0].VpcId
+		if skipByVPC[vpcUID] {
+			continue
+		}
+		vpc := res.GetVPC(vpcUID).(*commonvpc.VPC)
+		subnets := vpc.Subnets()
+		// todo - use only the public subnet:
+		if len(subnets) == 0 {
+			logging.Warnf("skipping internet gateway %s - it does not have any attached subnet\n", igwName)
+			continue
+		}
+		routerIgw := newIGW(igwName, igwID, subnets, vpc)
+		// TODO - where to put this resource?
```

Comment:
what does this TODO mean?

Reply:
removed the todo comment 
(it was not clear if the itg was per region or per vpc, )